platform/logging/appenders/console/ConsoleAppender.ts

```diff
@@ -0,0 +1,14 @@
+import { LogRecord } from '../../LogRecord';
+import { BaseAppender } from '../base/BaseAppender';
+import { ConsoleAppenderConfig } from './ConsoleAppenderConfig';
+
+export class ConsoleAppender extends BaseAppender {
+  constructor(protected readonly config: ConsoleAppenderConfig) {
+    super(config)
+  }
+
+  append(record: LogRecord) {
+    super.append(record);
+    console.log(this.logRecordToFormattedString(record));
```

Comment:
I want control and that looks like it'll give us control in the appender. I'd rather extract a "formatDate" util helper than using lots of `protected` methods (I prefer abstractions where I keep control over those that only DRY up the code)